test/workers.jl

```diff
@@ -17,6 +17,8 @@ using Test
     @testset "clean shutdown ($w)" begin
         close(w)
         @test !process_running(w.process)
+        @test w.process.termsignal == Base.SIGTERM
```

Comment:
I'm wondering where the "SIGTERM" comes from? Your PR description says:

> terminate workers with SIGTERM

but i think if it's a graceful close, the process just _exits_, meaning there is no signal at all.
So maybe this should be:
```suggestion
        @test w.process.termsignal == 0
```
?

That's what i'm seeing when running the tests:
```
  Expression: w.process.termsignal == Base.SIGTERM
   Evaluated: 0 == 15
```